src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1355,13 +1372,15 @@ export class PostgresStorageAdapter implements StorageAdapter {
 
     const qs = `SELECT ${columns} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern}`;
     debug(qs, values);
-    return this._client.any(qs, values)
-      .catch(error => {
+    // @flow-disable-next
```

Comment:
please remove :) we try to enforce flow in ther.

Reply:
I keep getting `Missing type annotation for U.` Don't know how to resolve it.